gpiozero/boards.py

```diff
@@ -2237,3 +2237,98 @@ def off(self):
         """
         self.buzzer.value = None
         super(JamHat, self).off()
+
+class Pibrella(CompositeOutputDevice):
+    """
+    Extends :class:`CompositeOutputDevice` for the `Cyntech/Pimoroni Pibrella`_ board.
+
+    There are 3 LEDs, one button, a tonal buzzer, 4 outputs (with LEDs) and 4 inputs. The pins are fixed.
+    Usage::
+
+        from gpiozero import Pibrella
+
+        pb = Pibrella()
+
+        pb.lights.on()
+        pb.button.wait_for_press()
+        pb.outputs.on()
+        pb.buzzer.play('A4')
+        pb.inputs.a.wait_for_press()
+        pb.off()
+
+    :param bool pwm:
+        If :data:`True`, construct :class: PWMLED instances to represent each
+        LED on the board. If :data:`False` (the default), construct regular
+        :class:`LED` instances.
+
+    :type pin_factory: Factory or None
+    :param pin_factory:
+        See :doc:`api_pins` for more information (this is an advanced feature
+        which most users can ignore).
+
+    .. _Pibrella: https://www.pibrella.com/
```

Comment:
I dunno if it's just me, but https://www.pibrella.com/ shows me a totally different website to http://www.pibrella.com/ ?! :confused:

Reply:
Ah, my bad, force of habit to https everything these days.

Reply:
Fixed